S21/osamaibraheem11/s21.py

```diff
@@ -0,0 +1,26 @@
+import csv
+
+vowels = ["A","E","I","O","U","Á","É","Í","Ó","Ú"]
+
+def LowerCase(word,language): 
+  if ('ga' in language) and (word.startswith('n') or word.startswith('t')) and word[1:2] in vowels:
```

Comment:
Testing with "in" here will pick up languages other than Irish, since the "ga" could appear anywhere in the string. For example "nga" is the language code for Ngbaka.